examples/json-access/json_access.description

```diff
@@ -0,0 +1,5 @@
+// Ballerina defines certain types as lax types, for which static typing rules are less strict.
+// For example field access (`.`) and optional field access (`?.`), which are generally allowed on
```

Comment:
For example, field access (`.`) and optional field access (`?.`), which are generally allowed on